lib/template/swift_source.rb

```diff
@@ -78,6 +78,53 @@ class Template::SwiftSource < Template::Base
     |<%- end -%>
     |}
     |<%- end -%>
+    |
+    |/** Dispatchers **/
+    |
+    |protocol Dispatcher {
+    |    func dispatch(_: URL)
+    |}
+    |
+    |extension Dispatcher {
```

Comment:
This code (and anything else that doesn't rely on variable post message data) will be moved to a separate file in the future. For now, keeping everything together makes testing this a lot easier for me.